website/docs/docs/building-a-dbt-project/building-models/using-custom-databases.md

```diff
@@ -1,6 +1,5 @@
 ---
-title: "Using databases"
-id: "using-custom-database"
+title: "Using custom databases"
 ---
```

Comment:
Did you drop the id by accident or is it an unneeded field?

Reply:
It's unneeded — when we moved from readme.io to hosting docs the script we used put these in, without it defaults back to the file name :)